From 3f8a1910c55e2be2edd74ac8c8c34349ca52d5e7 Mon Sep 17 00:00:00 2001
From: Yi Kong <yikong@google.com>
Date: Sun, 5 May 2019 02:49:35 -0700
Subject: [PATCH 5/6] sqlite: Turn off the new pass manager

Some apps crash if libsqlite is built with the new pass manager.

Change-Id: Ibd2da184fa614ce501c0994446dcb22e93dc4d1b
Signed-off-by: mydongistiny <jaysonedson@gmail.com>
---
 dist/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dist/Android.bp b/dist/Android.bp
index 2211d1f..dcae7ad 100644
--- a/dist/Android.bp
+++ b/dist/Android.bp
@@ -39,6 +39,7 @@ cc_defaults {
         "-DSQLITE_ENABLE_BATCH_ATOMIC_WRITE",
         "-Wno-unused-parameter",
         "-Werror",
+        "-fno-experimental-new-pass-manager",
     ],
 
     target: {
-- 
2.17.1

